libmbp/devices/pantech.cpp

```diff
@@ -39,6 +39,27 @@ namespace mbp
         device->setBootBlockDevs({ QCOM_BOOT, "/dev/block/mmcblk0p7" });
         device->setRecoveryBlockDevs({ QCOM_RECOVERY, "/dev/block/mmcblk0p14" });
         device->setExtraBlockDevs({ QCOM_TZ, "/dev/block/mmcblk0p8" });
+	device->twOptions()->supported = true;
+    	device->twOptions()->flags = 
+		Device::FLAG_TW_QCOM_RTC_FIX | Device::FLAG_TW_GRAPHICS_FORCE_USE_LINELENGTH;
+    	device->twOptions()->pixelFormat = Device::TwPixelFormat::RGBX_8888;
+    	device->twOptions()->brightnessPath = "/sys/class/leds/lcd-backlight/brightness";
+    	device->twOptions()->maxBrightness = 255;
+    	device->twOptions()->defaultBrightness = 160;
+	devices->push_back(device);
+
+	// PANTECH VEGA IRON
+        device = new Device();
+        device->setId("PANTECH");
```

Comment:
Please don't use the same ID ("PANTECH") for multiple devices as it will break certain things in the app. Your previous "ef63" and "ef52" IDs were good.